### PR TITLE
feat: updating homepage with wista video section

### PR DESCRIPTION
**Pull request by sethfeingold**
Screenshot 2023-02-07 at 8 29 29 AM

### CODE REVIEW

**Comment by sethfeingold** [comment]
Simple update to add video section to homepage, figma located [here](https://www.figma.com/file/V8VgVuNptyacVmRkOTITUP/rd-1?node-id=1508%3A2107&t=nwpSw0pC3Dz0EnR6-0)

File: frontend/next.config.js [hunk 1]
```diff
@@ -10,9 +10,17 @@ const isProdBuild = process.env.NODE_ENV === nodeEnv.PRODUCTION;
 
 const PLAUSIBLE_URL = "https://plausible.io";
 
-const TWITTER_URL = "https://cdn.syndication.twimg.com platform.twitter.com";
+const TWITTER_URL = ["https://cdn.syndication.twimg.com platform.twitter.com"];
```
Comment (tihuan) [inline frontend/next.config.js:13]: @sethfeingold what's the reason behind changing this link to an array? Thanks!

**Review by tihuan — COMMENTED** [review]
Looking great! Thanks for adding the video, @sethfeingold 🙏 Just one question otherwise LGTM thank you!

**Comment by tihuan** [comment]
Also a PR to merge into this branch 🙏 

PTAL @sethfeingold  https://github.com/chanzuckerberg/single-cell-data-portal/pull/4153

File: frontend/next.config.js (hunk 1 above)
Comment (sethfeingold) [reply frontend/next.config.js:13]: Ahh, can revert that! I think I was originally playing around locally with creating an array of url's (wistia, twitter, and plausible) and passing them to SCRIPT_SRC array, but chose to go back to keeping url's as separately declared strings. Left the brackets behind.

Comment (tihuan) [reply frontend/next.config.js]: Got it that makes sense! Thanks for reverting 🙌 LGTM!!

**Review by tihuan — APPROVED** [review]

**Pull request merged**